templates/artist/artist_detail.html

```diff
@@ -200,6 +226,69 @@ <h1 class="header-font">{{ artist.name }}</h1>
 {% block extrajs %}
     <script src="https://checkout.stripe.com/checkout.js"></script>
     <script type="text/javascript">
+
+    <script type="text/javascript" src="{% static "js/vendor/smooth-scroll.js" %}"></script>
+    <script type="text/javascript">
+
+
+
+        var doughnutData = [
+                {
+                    value: {{ campaign.fans_percentage }},
+                    color: "#46BFBD",
+                    highlight: "#5AD3D1",
+                    label: "Fan %"
+                },
+                {
+                    value: {{ campaign.artist_percentage }},
+                    color:"#F7464A",
+                    highlight: "#FF5A5E",
+                    label: "Artist %"
+                }
+
+            ];
+
+        var randomScalingFactor = function(){ return Math.round(Math.random()*100)};
+        var lineChartData = {
+            labels : ["1x","2x","3x","4x","5x"],
+            datasets : [
+                {
+                    label: "Artist Revenue",
+                    fillColor : "rgba(220,220,220,0.2)",
+                    strokeColor : "rgba(220,220,220,1)",
+                    pointColor : "rgba(220,220,220,1)",
+                    pointStrokeColor : "#fff",
+                    pointHighlightFill : "#fff",
+                    pointHighlightStroke : "rgba(220,220,220,1)",
+                    data : [{{ campaign|percentage_roi:100 }},{{ campaign|percentage_roi:200 }},{{ campaign|percentage_roi:400 }},{{ campaign|percentage_roi:1000 }},{{ campaign|percentage_roi:2000 }}]
+                },
+                {
+                    label: "Investor income",
+                    fillColor : "rgba(151,187,205,0.2)",
+                    strokeColor : "rgba(151,187,205,1)",
+                    pointColor : "rgba(151,187,205,1)",
+                    pointStrokeColor : "#fff",
+                    pointHighlightFill : "#fff",
+                    pointHighlightStroke : "rgba(151,187,205,1)",
+                    data : [{{ campaign|percentage_roi:50 }},{{ campaign|percentage_roi:100 }},{{ campaign|percentage_roi:200 }},{{ campaign|percentage_roi:500 }},{{ campaign|percentage_roi:1000 }}]
+                }
+            ]
+
+        }
+
+
+        window.onload = function(){
+            var ctxchart = document.getElementById("chart-area").getContext("2d");
+            window.myDoughnut = new Chart(ctxchart).Doughnut(doughnutData, {
+                responsive : true
+            });
+            var ctxline = document.getElementById("canvas").getContext("2d");
+            window.myLine = new Chart(ctxline).Line(lineChartData, {
+                responsive: true
+            });
+        }
+
+
         var perdiem_fee = {{ PERDIEM_FEE }};
```

Comment:
You should move these three lines to the top, wrapping all of the JavaScript code you added in the `{% if campaign %}` tag. If the artist does not have a campaign, their page breaks since you're requesting values like `fans_percentage` and `percentage_roi`, which have no meaning in that case.